compatibility/bazel_tools/daml_trigger/daml_trigger.bzl

```diff
@@ -230,5 +229,5 @@ chmod +x $(OUTS)
         server_args = server_args,
         server_files = server_files,
         server_files_prefix = server_files_prefix,
-        tags = extra_tags(compiler_version, runner_version) + ["exclusive"],
+        tags = ["exclusive"],
```

Comment:
I don't see how this disables the test, could you please explain?

Reply:
I just revert complety #15334, as both test seem flaky

Reply:
ah got it, `extra_tags` was doing the trick